back/lib/gem_extensions/omni_auth/strategies/open_id_connect.rb

```diff
@@ -4,12 +4,26 @@ module GemExtensions
   module OmniAuth
     module Strategies
       module OpenIdConnect
-        # Patch +OmniAuth::Strategies::OpenIDConnect+ to allow dynamic specification of the issuer.
+        # Patch +OmniAuth::Strategies::OpenIdConnect+ to allow dynamic specification of the issuer.
         def issuer
           return options.issuer.call(env) if options.issuer.respond_to?(:call)
 
           super
         end
+
```

Comment:
Because this is patching the `OpenIDConnect` gem I have made sure to test all the other open ID methods to ensure they still work